File: curso POO uber/JS/Driver.js
class Driver extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)
    }

    print_data_user(){
        console.log(`Name: ${this.name}`);
        console.log(`Document: ${this.document}`);
        console.log(`Email: ${this.email}`);
        console.log(`Password: ${this.password}`);
    }
}

File: curso POO uber/Java/Driver.java
class Driver extends Account{

    String number_p;

    public Driver (String name,String document,String email,String password, String number_p){
        super(name, document, email, password);
        this.number_p = number_p;
    }
    void print_data_driver(){
        System.out.println(" | Name driver: "+ name + " | Document driver:" + document + " | E-mail: " + email + " | Password: " + password);
    } 
}